dataset/test/binned_arithmetic_test.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include "scipp/dataset/bins.h"
 #include "scipp/dataset/dataset.h"
+#include "scipp/dataset/string.h"
```

Comment:
Is there some "operator<<" requirement here?

Reply:
Well, if a test fails it gets picked up by `gtest`, so yes and no?